packages/cover-traffic-daemon/package.json

```diff
@@ -13,8 +13,7 @@
     "test": "NODE_OPTIONS=\"--trace-warnings --unhandled-rejections=strict\" mocha --reporter=tap --full-trace",
     "coverage": "nyc --reporter=html mocha",
     "dev": "yarn clean && tsc -w",
-    "build": "yarn clean && tsc -p",
-    "prepublishOnly": "tsc -p ./tsconfig.npm.json",
```

Comment:
Why is this not needed anymore?

Reply:
Yarn 2+ deviates from npm by using different names for its lifecycle scripts. The optimization that built tests are not included in NPM packages now lives in a different PR to make sure that cleanup PR is independent from it.